easybuild/framework/easyblock.py

```diff
@@ -1014,6 +1014,25 @@ def make_devel_module(self, create_in_builddir=False):
         # cleanup: unload fake module, remove fake module dir
         self.clean_up_fake_module(fake_mod_data)
 
+    def make_module_deppaths(self):
+        """
+        Add specific 'module use' actions to module file, in order to find
+        dependencies outside the end user's MODULEPATH.
+        """
+        deppaths = self.cfg['moddependpaths']
+        if not deppaths:
+            return ''
+        elif not isinstance(deppaths, (str, tuple, list)):
+            raise EasyBuildError("moddependpaths value %s (type: %s) is not a string or collection",
```

Comment:
Sorry to be nit-picky here but `collection` is a loaded term ([at least in Lmod](https://lmod.readthedocs.io/en/latest/010_user.html#user-collections)), maybe just say `string, list or tuple`?

Reply:
Thanks.  I was hoping to be concise, and have one word for "a bunch of things."